python-backend/wolfram_client.py

```diff
@@ -66,16 +66,21 @@ def limit(expression: str) -> str:
             logger.error("Failed to obtain Wolfram API result", e)
 
     @staticmethod
-    def closed_form(expression: str) -> str:
+    def closed_form(expression: str) -> dict[str, str]:
         """
         Query the Wolfram results API for limits of a string expression
         :param expression: Mathematical expression for which we would like to compute the limit(s)
         :return: Wolfram API response "subpods" with the computed limit(s)
         """
         try:
             result = WolframClient.ask(query=expression, include_pod="PossibleClosedForm")
+            print(result)
```

Comment:
Is this an artifact left over from earlier debugging? If not, consider replacing print with a logger.

Reply:
thanks for catching that! 🎣